docs/common-bugs.rst

```diff
@@ -118,7 +118,7 @@ Changes in package groups and environments
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 :Issue: The reporter wants a new package to be installed by default.
-:Solution: Reassigning to comps.
+:Solution: Reassigning to comps or distribution.
```

Comment:
The commit summary says to use "distribution instead of comps", but the change suggests to use "comps or distribution". I (as a lazy user) would pick the first listed component.
In addition to that, https://fedoraproject.org/wiki/How_to_use_and_edit_comps.xml_for_package_groups#All_other_groups says to use the comps component - it looks like the current text is correct.

Reply:
The `comps` component doesn't seem to exist in Bugzilla anymore and the bug, that we needed to recently reassign, ended up in the `distribution` component. There is only the `fedora-comps` repository at https://pagure.io/fedora-comps with their own issue tracker. It seemed better to keep there all variants in case that this will change again in the future, but I can change it.

Reply:
Since the `comps` component cannot be selected when creating a new bug, it should not be mentioned in the documentation - we do not want to confuse the users.
I've asked for clarification and updating the "How to use and edit comps.xml for package groups" wiki page:
https://lists.fedoraproject.org/archives/list/packaging@lists.fedoraproject.org/thread/KTZW557TLXXNEDSEBJFR77APHUSDIYOT/